### PR TITLE
[4.4] Phan: Fixes to com_finder

### PR DESCRIPTION
**Pull request by Hackwar**
### Summary of Changes
This fixes some issues phan reported and I removed some deprecated code.

### Testing Instructions
Codereview

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [X] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [X] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: components/com_finder/src/Model/SearchModel.php [hunk 1]
```diff
@@ -173,11 +173,13 @@ protected function getListQuery()
             ->where('l.published = 1');
 
         // Get the current date, minus seconds.
-        $nowDate = $db->quote(substr_replace(Factory::getDate()->toSql(), '00', -2));
+        $nowDate = Factory::getDate()->toSql();
```
Comment (wilsonge) [inline components/com_finder/src/Model/SearchModel.php]: Please let's not do this as part of a code style pr. i accept it's terrible. but this needs testing and stops us merging on code review.

Comment (richard67) [reply components/com_finder/src/Model/SearchModel.php]: I agree. This change should be made with another PR.

Comment (Hackwar) [reply components/com_finder/src/Model/SearchModel.php]: done

**Comment by laoneo** [comment]
Thanks!

**Pull request merged**